src/App.tsx

```diff
@@ -15,47 +16,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [reverse, setReverse] = useState(false);
```

Comment:
It is boolean variable so better name is 
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```